worker/src/main/java/gov/cms/ab2d/worker/processor/PatientContractCallable.java

```diff
@@ -0,0 +1,103 @@
+package gov.cms.ab2d.worker.processor;
+
+import gov.cms.ab2d.bfd.client.BFDClient;
+import gov.cms.ab2d.worker.processor.domainmodel.ContractMapping;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.exception.ExceptionUtils;
+import org.hl7.fhir.dstu3.model.Bundle;
+import org.hl7.fhir.dstu3.model.Identifier;
+import org.hl7.fhir.dstu3.model.Patient;
+import org.hl7.fhir.dstu3.model.ResourceType;
+
+import java.util.Set;
+import java.util.concurrent.Callable;
+import java.util.stream.Collectors;
+
+@Slf4j
+public class PatientContractCallable implements Callable<ContractMapping> {
+    private static final String BENEFICIARY_ID = "https://bluebutton.cms.gov/resources/variables/bene_id";
+    private int month;
+    private String contractNumber;
+    private BFDClient bfdClient;
+
+    public PatientContractCallable(String contractNumber, int month, BFDClient bfdClient) {
+        this.month = month;
+        this.contractNumber = contractNumber;
+        this.bfdClient = bfdClient;
+    }
+
+    @Override
+    public ContractMapping call() throws Exception {
+        System.out.println(Thread.currentThread().getName() + "-" + Thread.currentThread().getId());
```

Comment:
This should be a log statement or just remove if it was for debugging.

Reply:
I wanted an easy to check in the log when jobs start and end so that we can see it's doing them in parallel. I can remove it later. During some testing, I had issues with things going one by one, not sure why, but this way, we can make sure that's not happening..